### ISSUE TITLE
Index create failed on dataset sift-50m

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#5449]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
on standalone mode:
```

benchmark-gtlt8-3244100830: 2021-05-27 03:47:12,462 - milvus_benchmark.client:49 - DEBUG - Milvus insert run in 3.79s
benchmark-gtlt8-3244100830: 2021-05-27 03:47:12,465 - milvus_benchmark.client:364 - DEBUG - Row count: 49950000 in collection: <sift
_50m_128_l2>
benchmark-gtlt8-3244100830: 2021-05-27 03:47:12,466 - milvus_benchmark.runners.base:87 - DEBUG - 49950000
benchmark-gtlt8-3244100830: 2021-05-27 03:47:12,467 - milvus_benchmark.runners.base:149 - INFO - {'total_time': 3942.47, 'rps': 1268
2.42, 'ni_time': 3.94}
benchmark-gtlt8-3244100830: 2021-05-27 03:47:18,610 - milvus_benchmark.client:49 - DEBUG - Milvus flush run in 6.05s
benchmark-gtlt8-3244100830: 2021-05-27 03:47:18,614 - milvus_benchmark.client:364 - DEBUG - Row count: 50000000 in collection: <sift
_50m_128_l2>
benchmark-gtlt8-3244100830: 2021-05-27 03:47:18,615 - milvus_benchmark.runners.search:248 - DEBUG - 50000000
benchmark-gtlt8-3244100830: 2021-05-27 03:47:18,615 - milvus_benchmark.runners.search:250 - DEBUG - Start build index for last file
benchmark-gtlt8-3244100830: 2021-05-27 03:47:18,616 - milvus_benchmark.client:236 - INFO - Building index start, collection_name: si
ft_50m_128_l2, index_type: IVF_FLAT, metric_type: L2
benchmark-gtlt8-3244100830: 2021-05-27 03:47:18,617 - milvus_benchmark.client:238 - INFO - {'nlist': 2048}
benchmark-gtlt8-3244100830: 2021-05-27 03:47:19,520 - milvus.client.grpc_handler:56 - ERROR - Error: <BaseException: (code=1, messag
e=CreateIndex failed, error = index id = 425219267717496833 exist)>
benchmark-gtlt8-3244100830: Error: <BaseException: (code=1, message=CreateIndex failed, error = index id = 425219267717496833 exist)
>
benchmark-gtlt8-3244100830: 2021-05-27 03:47:19,521 - milvus_benchmark.main:113 - ERROR - <BaseException: (code=1, message=CreateInd
ex failed, error = index id = 425219267717496833 exist)>
benchmark-gtlt8-3244100830: 2021-05-27 03:47:19,524 - milvus_benchmark.main:114 - ERROR - Traceback (most recent call last):
benchmark-gtlt8-3244100830:   File "main.py", line 88, in run_suite
benchmark-gtlt8-3244100830:     runner.prepare(**cases[0])
benchmark-gtlt8-3244100830:   File "/src/milvus_benchmark/runners/search.py", line 252, in prepare
benchmark-gtlt8-3244100830:     self.milvus.create_index(index_field_name, case_param["index_type"], case_param["metric_type"], inde
x_param=case_param["index_param"])
benchmark-gtlt8-3244100830:   File "/src/milvus_benchmark/client.py", line 46, in wrapper
benchmark-gtlt8-3244100830:     result = func(*args, **kwargs)
benchmark-gtlt8-3244100830:   File "/src/milvus_benchmark/client.py", line 244, in create_index
benchmark-gtlt8-3244100830:     self._milvus.create_index(self._collection_name, field_name, index_params, _async=_async)
benchmark-gtlt8-3244100830:   File "/usr/local/lib/python3.6/site-packages/milvus/client/stub.py", line 39, in inner
benchmark-gtlt8-3244100830:     return func(self, *args, **kwargs)
benchmark-gtlt8-3244100830:   File "/usr/local/lib/python3.6/site-packages/milvus/client/stub.py", line 728, in create_index
benchmark-gtlt8-3244100830:     return handler.create_index(collection_name, field_name, params, timeout, **kwargs)
benchmark-gtlt8-3244100830:   File "/usr/local/lib/python3.6/site-packages/milvus/client/grpc_handler.py", line 62, in handler
benchmark-gtlt8-3244100830:     raise e
benchmark-gtlt8-3244100830:   File "/usr/local/lib/python3.6/site-packages/milvus/client/grpc_handler.py", line 54, in handler
benchmark-gtlt8-3244100830:     return func(self, *args, **kwargs)
benchmark-gtlt8-3244100830:   File "/usr/local/lib/python3.6/site-packages/milvus/client/grpc_handler.py", line 808, in create_index
benchmark-gtlt8-3244100830:     raise BaseException(status.error_code, status.reason)
benchmark-gtlt8-3244100830: milvus.client.exceptions.BaseException: <BaseException: (code=1, message=CreateIndex failed, error = ind
ex id = 425219267717496833 exist)>
```
**Steps/Code to reproduce behavior**
1. create collection
2. create index: ivf-flat, nlist: 2048
3. insert entities, 50,000 entities per request
4. insert 50m entities, flush
5. create index again, error returned
**Expected behavior**

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Software conditions (OS, CPU, GPU, Memory)

- Milvus version (master or released version)

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**

**Additional context**

**Comment by del-zhenwu** [comment]
Client hang for two days this time.

**Comment by neza2017** [comment]
this issue fix by #6042, please check it

my test code
```python
from pymilvus import Milvus, IndexType, MetricType, DataType
from sklearn import preprocessing
import random
import numpy as np
from datetime import datetime

_HOST = '127.0.0.1'
_PORT = '19530'
_DIM = 128
_FILE_PREFIX = '/disk/sift1b/binary_128d_'
_NUM_FILES = 500
_STEP = 50000
_NUM_ROWS_PER_FILE = 100000

connection = Milvus(_HOST, _PORT)
collection_name = 'example_collection_'
ok = connection.has_collection(collection_name)
if ok:
    connection.drop_collection(collection_name)

param = {
    "fields": [
        {"name": "int64", "type": DataType.INT64, "is_primary": True},
        {
            "name": "float32",
            "type": DataType.FLOAT_VECTOR,
            "metric_type": "L2",
            "params": {"dim": _DIM},
            "indexes": [{"metric_type": "L2"}]
        }
    ],
    "auto_id": False,
}
connection.create_collection(collection_name, param)
ok = connection.has_collection(collection_name)
print("has collection", collection_name, ok)

count = 0
for file_idx in range(0, _NUM_FILES):
    file_name = _FILE_PREFIX + str(file_idx).zfill(5) + ".npy"
    array = np.load(file_name)
    for p in range(0, _NUM_ROWS_PER_FILE, _STEP):
        entities = [
            {"name": "int64", "type": DataType.INT64, "values": [i for i in range(count, count + _STEP)]},
            {"name": "float32", "type": DataType.FLOAT_VECTOR, "values": array[p:p + _STEP][:].tolist()}
        ]
        ids = connection.insert(collection_name, entities)
        count = count + _STEP
        print("insert count :", count)

connection.flush([collection_name])
stats = connection.get_collection_stats(collection_name)
print("total rows", stats["row_count"], datetime.now().strftime("%H:%M:%S"))

index_params = {
    "metric_type": "L2",
    "index_type": "IVF_FLAT",
    "params": {"nlist": 2048}
}

connection.create_index(collection_name,"float32",index_params)
print("crete index success", datetime.now().strftime("%H:%M:%S"))

````
output
```txt
insert count : 49800000
insert count : 49850000
insert count : 49900000
insert count : 49950000
insert count : 50000000
total rows 50000000 20:44:38
crete index success 22:01:45
```

**Comment by neza2017** [comment]
@del-zhenwu
